src/modules/roc_audio/sample_spec.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2020 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_audio/sample_spec.h
+
+#ifndef ROC_AUDIO_SAMPLE_SPEC_H_
+#define ROC_AUDIO_SAMPLE_SPEC_H_
+
+#include "roc_packet/units.h"
+
+namespace roc {
+namespace audio {
+
+//! Bitmask of channels present in audio packet.
+typedef uint32_t channel_mask_t;
+
+//! Default sample rate, number of samples per second.
+const size_t DefaultSampleRate = 44100;
+
+//! Default channel mask.
+const channel_mask_t DefaultChannelMask = 0x3;
+
+class SampleSpec {
+public:
+    SampleSpec();
+    SampleSpec(size_t sample_rate, channel_mask_t channels);
+
+    size_t getSampleRate() const;
+    void setSampleRate(size_t sample_rate);
+    channel_mask_t getChannels() const;
+    void setChannels(channel_mask_t channels);
```

Comment:
Some code style issues:

* Please use snake_case for method names.
* Could you rename getters from getFoo() to just foo()?
* Let's rename getChannels/setChannels to channel_mask()/set_channel_mask().
* Every public member should have Doxygen comment.
* Private methods should end with underscore: calc_num_channels -> calc_num_channels_.